src/cranbtree.c

```diff
@@ -110,7 +132,9 @@ cranbtree_t *cbt_clone(cranbtree_t * cbt)
 
 /**
   * cranbtree_t*, int, void* -> void
```

Comment:
Should be changed to return void* and return NULL on error, or the object that was just inserted on Success